README.md

```diff
@@ -415,3 +417,5 @@ module.exports = [
 
 [actions-image]: https://img.shields.io/endpoint?url=https://github-actions-badge-u3jn4tfpocch.runkit.sh/jsx-eslint/eslint-plugin-react
 [actions-url]: https://github.com/jsx-eslint/eslint-plugin-react/actions
+
+[badge-jsx-runtime]: https://img.shields.io/badge/-jsx--runtime-blue.svg
```

Comment:
I added a badge to represent the `jsx-runtime` config. We need a way to represent it now that we indicate which configs disable rules. 

![badge-jsx-runtime][]

[badge-jsx-runtime]: https://img.shields.io/badge/-jsx--runtime-blue.svg

Alternatively, we could choose another emoji to represent it with `--config-emoji jsx-runtime,😋`. Feel free to suggest an emoji to use, or we can just proceed like this for now.

Reply:
This is fine for now

Reply:
Follow-up: https://github.com/jsx-eslint/eslint-plugin-react/pull/3504